src/app/inventory/store/d2-item-factory.ts

```diff
@@ -658,6 +673,9 @@ export function makeItem(
 
   return createdItem;
 }
+export function isTrialsPassage(item: DimItem) {
+  return item.objectives?.length === 3 && item.typeName === 'Trials Passage';
```

Comment:
This will not work in other locales... in `de` typeName is `Prüfungen-Pass`

Reply:
Good point, I should have read the tooltip. I've pushed another change which goes about it in a different way.